website/settings/defaults.py

```diff
@@ -1958,3 +1959,21 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+STORAGE_WARNING_THRESHOLD = .9  # percent of maximum storage used before users get a warning message
+STORAGE_LIMIT_PUBLIC = 50
+STORAGE_LIMIT_PRIVATE = 5
+
+
+@enum.unique
+class StorageLimits(enum.IntEnum):
+    """
+    Values here are in GBs
+    """
+    DEFAULT = 0
+    OVER_PRIVATE = STORAGE_LIMIT_PRIVATE
```

Comment:
I would avoid using a raw value for these. Enums are generally best used as simple abstractions. An IntEnum feels worth using here to easily enforce a hierarchy, but this approach basically makes them constants with some extra support for pretty printing.

This is emphasized by your logic above to effectively ignore/rewrite the values in the case of a custom storage cap.